particle/apps/README.md

```diff
@@ -2,12 +2,15 @@
 
 ## Adding or Removing Apps
```

Comment:
I added a similar section to the Sprint 0 page. I think this can be removed / combined with that section. It feels better there since that's a "so you actually want to use this on your project" without muddying the apps readme.

Reply:
I think including instructions in how to remove or add apps is relevant to Apps on the whole, and not something that's only done in Sprint 0/project setup, and vote for keeping it here. We should update the Sprint 0 text to link to this section if we all agree.